projects/distributed-tracing/src/pages/trace-detail/trace-detail.page.component.ts

```diff
@@ -53,13 +50,16 @@ import { TraceDetails, TraceDetailService } from './trace-detail.service';
         </div>
       </div>
 
-      <ht-navigable-dashboard
-        class="scrollable-container"
-        [padding]="0"
-        navLocation="${traceDetailDashboard.location}"
-        (dashboardReady)="this.onDashboardReady($event)"
-      >
-      </ht-navigable-dashboard>
+      <ht-navigable-tab-group class="tabs">
+        <ht-navigable-tab path="sequence"> Sequence </ht-navigable-tab>
+        <ng-container *ngIf="this.logEvents$ | async as logEvents">
```

Comment:
why this ngIf? Is it feature flagged?

Reply:
Nope, but if we don't use this or use `*htLetAsync` , then this behaves differently.
If we have logEvents as `undefined` (which I do not think will be the case, apart from errors from backend) ,  I guess it's better to not show tabs. Which can be a indication of error.